### DISCUSSION TITLE
Is it possible to use StrainGE with MAGs?

### DISCUSSION THREAD

**Discussion by SilasK** [broadinstitute/StrainGE#22] [Q&A] [answered]
Is it possible to use StrainGE with MAGs or with genome assemblies that are not completely closed. e.g. they have multiple contigs.

**Comment by lrvdijk** [comment, marked as answer]
This is related to https://github.com/broadinstitute/StrainGE/discussions/27, and the answer is similar. In principle it should work, though there are more caveats.
